python/lsst/ts/imsim/opdMetrology.py

```diff
@@ -89,6 +78,8 @@ def setDefaultLsstWfsGQ(self):
         self.wt = np.array([1.0, 1.0, 1.0, 1.0])
         wfsFieldX, wfsFieldY, sensorIds = self.getDefaultLsstWfsGQ()
         self.fieldX, self.fieldY = (wfsFieldX, wfsFieldY)
+        # Convert from CCS to ZCS for current OFC
```

Comment:
Add TO-DO tag at the beginning of the comment?

Reply:
Done